apps/passport-client/components/screens/ProveScreen/GenericProveScreen.tsx

```diff
@@ -48,7 +48,7 @@ export function GenericProveScreen({ req }: { req: PCDGetRequest }) {
     <>
       <MaybeModal fullScreen />
       <AppContainer bg="gray">
-        <AppHeader>
+        <AppHeader isProveScreen={true}>
```

Comment:
let's call this prop `hideSubscriptionFolder` since that's more direct and can be generically used in other screens, e.g. adding a pcd

Reply:
Done.